src/mac/mod.rs

```diff
@@ -0,0 +1,21 @@
+//! Partial implementation of the IEEE 802.15.4 MAC layer
+
+mod frame;
+pub mod command;
+pub mod beacon;
+
+pub use frame::{
```

Comment:
Since this is a breaking change anyway, we might consider not re-exporting all of these (or any of these). Not sure.

Reply:
I do not have a strong opinion. I might prefer it this way, since I thing these enums and structs will be used the most.

Reply:
I don't feel strongly either. Let's leave it as is then, for now.